### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: apps/channel/templates/_path.jade [hunk 1]
```diff
@@ -25,9 +25,11 @@ block metadata
 
     include meta_pockets/connections
 
-    if user
+    if can.update
       include meta_pockets/privacy
       include meta_pockets/export
+
+    if can.manage
```
Comment (dzucconi) [inline apps/channel/templates/_path.jade]: woof

Comment (dzucconi) [reply apps/channel/templates/_path.jade]: I'd rather u say `if can.destroy`

Comment (broskoski) [reply apps/channel/templates/_path.jade]: why woof

Comment (broskoski) [reply apps/channel/templates/_path.jade]: Can do

Comment (dzucconi) [reply apps/channel/templates/_path.jade]: Not that the server would have let anyone delete anything... It just must have been disconcerting to see that button when you shouldn't.

**Pull request merged**